addons/skin.estuary/xml/DialogVideoInfo.xml

```diff
@@ -434,22 +434,22 @@
 							<control type="image">
 								<top>20</top>
 								<left>20</left>
-								<width>204</width>
-								<height>277</height>
+								<width>186</width>
+								<height>279</height>
 								<texture background="true">$INFO[ListItem.Art(poster)]</texture>
-								<aspectratio aligny="center">keep</aspectratio>
+								<aspectratio aligny="center">scale</aspectratio>
 							</control>
 							<control type="image">
-								<left>30</left>
-								<width>184</width>
+								<left>20</left>
+								<width>186</width>
 								<height>180</height>
 								<bottom>10</bottom>
 								<texture>overlays/overlayfade.png</texture>
```

Comment:
Do either of you know the purpose of this overlay?

It was nothing changed with this PR I noticed it renders some poster unreadable, for example this Terminator set

![image](https://github.com/xbmc/xbmc/assets/5781142/1538c422-968c-435a-b1ef-da566c7943b4)

Removing that overlay

![image](https://github.com/xbmc/xbmc/assets/5781142/476c8f52-02a9-4e20-ac00-8aff62cd1b78)

Reply:
Isn't this overly there to make symbols like the "already started watching this" symbol more visible on very bright posters? No idea if it should be shown unconditionally.